Automattic/jetpack
# AI Assistant: Add FeatureControl to Write Brief

Fixes #39167 

## Proposed changes:

* Uses the AI Helper to get feature availability from the backend
* Checks those values before enabling Write Brief or any individual highlight type

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this PR to your sandbox (testing locally alone won't work, as the AI helper class needs to be in sync with the backend)
* Go to the block editor
* Check that Breve is enabled and all its features (spelling is still under beta flag, check will be removed in a follow-up)
* Edit the value of `write-brief` or any of its subfeatures in D160005-code
* Check that the features are enabled or disabled according to the function

Only Write Brief checks for the function return value, other features will work regardless of their value.
